docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -0,0 +1,192 @@
+
+Example of customization:
+
+```javascript
+feedbackIntegration({
+  buttonLabel: "Feedback",
+  submitButtonLabel: "Send Feedback",
+  formTitle: "Send Feedback",
+});
+```
+
+### Style Customization
+
+You can customize placement of the feedback components on the widget, as well as the fonts and colors.
+
+The example below shows how to customize the submit button background color and border radius with the `feedbackIntegration`.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
+Sentry.feedbackIntegration({
+    styles:{
+        submitButton: {
+            backgroundColor: '#6a1b9a',
+            borderRadius: 5,
+        },
+    },
+});
+
+Sentry.showFeedbackForm();
+```
+
+The same can be achived by passing the `styles` prop to the `FeedbackForm` component:
+
+```javascript
+import { FeedbackForm } from "@sentry/react-native";
+
+<FeedbackForm
+  styles={{
+    submitButton: {
+      backgroundColor: '#6a1b9a',
+      borderRadius: 5,
+    },
+  }}
+/>
+```
+
+The following styles are available for customisation.
+
+| Style              | Type         | Description                                                                      |
+| ------------------ | ------------ | -------------------------------------------------------------------------------- |
+| `container`        | `ViewStyle`  | The form container style.                                                        |
+| `title`            | `TextStyle`  | The title text style.                                                            |
+| `label`            | `TextStyle`  | The label text style (name, email).                                              |
+| `input`            | `TextStyle`  | The input field text style (name, email).                                        |
+| `textArea`         | `TextStyle`  | The message text style.                                                          |
+| `submitButton`     | `ViewStyle`  | The submit button style.                                                         |
+| `submitText`       | `TextStyle`  | The submit button text style.                                                    |
+| `cancelButton`     | `ViewStyle`  | The cancel button style.                                                         |
+| `cancelText`       | `TextStyle`  | The cancel button text style.                                                    |
+| `screenshotButton` | `ViewStyle`  | the screenshot button style.                                                     |
+| `screenshotText`   | `TextStyle`  | The screenshot button text style.                                                |
+| `titleContainer`   | `ViewStyle`  | The title container style.                                                       |
+| `sentryLogo`       | `ImageStyle` | The Sentry branding logo style.                                                  |
```

Comment:
I would leave the `sentryLogo` style from the docs, just to avoid users messing with our branding.